near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -14,8 +14,9 @@ use near_sdk::{
 };
 use std::collections::HashMap;
 
-const GAS_FOR_RESOLVE_TRANSFER: Gas = 5_000_000_000_000;
-const GAS_FOR_FT_TRANSFER_CALL: Gas = 25_000_000_000_000 + GAS_FOR_RESOLVE_TRANSFER;
+const GAS_FOR_RESOLVE_TRANSFER_INT: u64 = 5_000_000_000_000;
+const GAS_FOR_RESOLVE_TRANSFER: Gas = Gas::new(GAS_FOR_RESOLVE_TRANSFER_INT);
+const GAS_FOR_FT_TRANSFER_CALL: Gas = Gas::new(25_000_000_000_000 + GAS_FOR_RESOLVE_TRANSFER_INT);
```

Comment:
if only we had `#![feature(const_trait_impl)]`

Reply:
We might provide inherent const methods: 

```rust
const GAS_FOR_RESOLVE_TRANSFER Gas = Gas(5_000_000_000_000);
const GAS_FOR_FT_TRANSFER_CALL: Gas = Gas::new(25_000_000_000_000).add(GAS_FOR_RESOLVE_TRANSFER_INT);
```

where `add` is defined as

```rust
impl Gas {
    pub const fn add(self, other: Gas) -> Gas { Gas(self.0 + other.0)}
}

impl ops::Addfor Gas {
    fn add(self, other: Gas) -> Gas { self.add(other) }
}
```

Reply:
I was considering that, but seemed like a smell to have the trait impl and const fn have the same name, and if it wasn't it would still be an unintuitive usage for most developers. I think if we are exposing the inner type, and I'll do that, it somewhat removes the need for this